guide/src/building_and_distribution.md

```diff
@@ -1,58 +1,91 @@
 # Building and Distribution
 
-## Python version
+This chapter of the guide goes into detail on how to build and distribute projects using PyO3. The way to achieve this is very different depending on whether the project is a Python module implemented in Rust, or a Rust binary embedding Python. For both types of project there are also common problems such as the Python version to build for and the [linker](https://en.wikipedia.org/wiki/Linker_(computing)) arguments to use.
+
+The material in this chapter is intended for users who have already read the PyO3 [README](#index.md). It covers in turn the choices that can be made for Python modules and for Rust binaries. There is also a section on the end about cross-compiling projects using PyO3.
```

Comment:
Non-native-speaker question: is there a semantic difference between "at the end" and "on the end"?

Reply:
Heh yes, I'd use "at the end" to describe a location of something, e.g. "at the end of the document" or "at the end of the path", versus "on the end" to describe 

This is a good spot and I think a typo by me, because I would actually use "at the end" in this sentence. I think I started writing something else and then didn't correct the start when I changed tack.

```suggestion
The material in this chapter is intended for users who have already read the PyO3 [README](#index.md). It covers in turn the choices that can be made for Python modules and for Rust binaries. There is also a section at the end about cross-compiling projects using PyO3.
```